library/src/options.cxx

```diff
@@ -136,6 +136,31 @@ std::pair<std::string, unsigned int> options::getClosestOption(const std::string
   return ret;
 }
 
+//----------------------------------------------------------------------------
+bool options::isOptional(const std::string& option) const
+{
+  return options_tools::isOptional(option);
+}
+
+//----------------------------------------------------------------------------
+void options::reset(const std::string& name)
+{
+  options_tools::reset(*this, name);
+}
+
+//----------------------------------------------------------------------------
+void options::removeValue(const std::string& name)
+{
+  if (isOptional(name))
```

Comment:
this->isOptional

Reply:
why is there a need to explicitly use `this->`